### ISSUE TITLE
[BUG] - [Dungeon Crawl Classics] - Lootable token opens as a Vault

### ISSUE THREAD

**Issue by Necrotech242** [fantasycalendar/FoundryVTT-ItemPiles#319]
**Have you tried to reset Item Piles' system settings?**
Yes

**Describe the bug**
When I use the macro Make tokens lootable it opens as a Vault window and not as an Item Pile

**To Reproduce**

1. Click on token
2. Use Make tokens lootable-macro
3. Double click (as GM or Player)
4. Opens as a Vault Window

**Expected behavior**
Opens as a Item pile

**Screenshots**
![Skärmklipp](https://user-images.githubusercontent.com/90268107/228042573-b65099b7-ee76-471f-94c3-981a895f1ba0.JPG)

**Setup:**

- Foundry Version: 10 Build 291
- System Version: Dungeon Crawl Classics 0.35.1
- Item Piles Version: 2.5.7
- Browser & version 111.0.5563.147 (Officiell version) (64 bit)

**Active modules:**

- Item Piles
- libwrapper
- socketlib
- I have disabled all other modules

**Comment by Haxxer** [comment]
Is your default item pile actor set to vault? That would be the cause.

**Comment by Necrotech242** [comment]
Yes that is correct. Must have mixed things up. Many thanks for the quick response.
